README.md

```diff
@@ -274,8 +274,21 @@ ctest -R name-of-test-to-run
 # Documentation and support
 
 For more informaton, consult QMCPACK pages at http://www.qmcpack.org,
-the linked documentation, and the local copy of the manual in
-manual/qmcpack_manual.pdf
+the linked documentation, https://docs.qmcpack.org/qmcpack_manual.pdf .
```

Comment:
This is not clear to me what 'linked documentation' is referring to.  Does it mean the manual, or is it a reference to some other documentation?